Assets/MixedRealityToolkit/_Core/Devices/OpenVR/GenericOpenVRController.cs

```diff
@@ -12,5 +16,335 @@ public class GenericOpenVRController : BaseController
     {
         public GenericOpenVRController(TrackingState trackingState, Handedness controllerHandedness, IMixedRealityInputSource inputSource = null, MixedRealityInteractionMapping[] interactions = null)
                 : base(trackingState, controllerHandedness, inputSource, interactions) { }
+
+        /// <summary>
+        /// The current source state reading for this OpenVR Controller.
+        /// </summary>
+        public XRNodeState LastStateReading;
+
+        private Vector3 currentControllerPosition = Vector3.zero;
+        private Quaternion currentControllerRotation = Quaternion.identity;
+
+        private Vector3 currentPointerPosition = Vector3.zero;
+        private Quaternion currentPointerRotation = Quaternion.identity;
+        private MixedRealityPose currentPointerData = new MixedRealityPose(Vector3.zero, Quaternion.identity);
```

Comment:
`MixedRealityPose.ZeroIdentity`?

Reply:
Updated. Will need a separate PR to update the WMRcontrollers this was copied from

Reply:
I had thought I addressed this already in the WMR class

Reply:
Just checked and we had, I must have simply copied it before that change :D